jot/JotTextView.m

```diff
@@ -8,6 +8,9 @@
 
 #import "JotTextView.h"
 
+static const CGFloat kTextMargin = 10.f;
+static const CGFloat kCornerRadius = 8.f;
```

Comment:
Was there a design spec that made the corners rounder? I would've thought that we would try to reduce/remove rounded corners as part of our new look.

Reply:
Yep - new designs use 8px cornerRadius for text annotations

Reply:
Is this from Scott's new Lens Polish updates?

Reply:
The mocks are a lie! Checked with Scott and we're going back to 5px